book/src/csi-driver-object.md

```diff
@@ -70,6 +75,19 @@ There are four important fields:
     The default is `Persistent`, which is the normal PVC/PV mechanism. `Ephemeral` enables
     [inline ephemeral volumes](ephemeral-local-volumes.md) in addition (when both
     are listed) or instead of normal volumes (when it is the only entry in the list).
+* `tokenRequests`
+  * This field was added in Kubernetes 1.20 and cannot be set when using an older Kubernetes release.
+  * This field is alpha.
```

Comment:
Can you specify the feature gate that is required as well?

Reply:
done